curvefs/src/mds/topology/topology_manager.cpp

```diff
@@ -632,6 +632,9 @@ void TopologyManager::CreatePartitions(const CreatePartitionRequest *request,
     auto partitionInfoList = response->mutable_partitioninfolist();
     response->set_statuscode(TopoStatusCode::TOPO_OK);
 
+    // get lock and avoid multiMountpoint create concurrently
+    NameLockGuard lock(createPartitionMutex_, std::to_string(fsId));
+
     while (partitionInfoList->size() < count) {
```

Comment:
> uint32_t pNumber = topology_->GetPartitionNumberOfFs(fsId);
> uint64_t idStart = pNumber * option_.idNumberInPartition;
> uint64_t idEnd = (pNumber + 1) * option_.idNumberInPartition - 1;

if delete partition, the start and end will duplicate.
fix it or add a todo here.

Reply:
This will fix at another pr later.